### PR TITLE
configure backend connection pool settings via CR

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -29,6 +29,14 @@ skipper_clusterratelimit: "false"
 skipper_clusterratelimit: "true"
 {{end}}
 
+# skipper backend timeout defaults
+skipper_expect_continue_timeout_backend: "30s"
+skipper_keepalive_backend: "30s"
+skipper_max_idle_connection_backend: 0
```
Comment (mikkeloscar) [inline cluster/config-defaults.yaml]: I wonder if this is parsed correctly and doesn't end up as empty string or something like that. I would expect this should be a string to be correct but I'm not 100% sure. @aermakov-zalando do you know?

Comment (szuecs) [reply cluster/config-defaults.yaml]: `func Itoa(i int) string` should be fine in Go, there is no error as return value

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**